packages/aws-cdk/test/cdk-toolkit.test.ts

```diff
@@ -390,20 +390,6 @@ describe('synth', () => {
     await expect(toolkit.synth(['Test-Stack-A'], false, true)).resolves.toBeUndefined();
   });
 
-  describe('post-synth validation', () => {
```

Comment:
What's going on here?

Reply:
This was an empty describe block (i.e no tests in it) - jest now rejects these. Good callout.